SourceryTests/Models/VariableSpec.swift

```diff
@@ -25,6 +25,16 @@ class VariableSpec: QuickSpec {
 
             it("reports optional false") {
                 expect(sut?.isOptional).to(beFalse())
+                sut?.typeName = "Int?"
```

Comment:
it's better to use context and separate tests scenarios as it makes understanding regressions and code easier. So here I'd change this into:
```swift
context("given an optional type") {
  it("recognizes optional) {
    expect(sut?.isOptional).to(beFalse())
  }
  it("returns non-optional type for unwrappedTypeName") {
    expect(sut?.unwrappedTypeName).to(equal("Int"))
  }
}
```

and another context for non-optional type